catalog/task/validate-single-component/0.1/README.md

```diff
@@ -0,0 +1,10 @@
+# validate-single-component
+
+Tekton task to validates that the snapshot
+only contains a single component A pipeline should fail otherwise.
```

Comment:
the comma was deleted but no period was added

Reply:
Fixed, thanks!